frontend/src/metabase/admin/app/components/AdminApp/AdminApp.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from "react";
+import DeprecationBanner from "../../containers/DeprecationNotice";
```

Comment:
Probably, you missed this import when renamed the component `DeprecationBanner ` -> `DeprecationNotice`

Reply:
Yep, fixed